logxhost/service.go

```diff
@@ -75,3 +75,11 @@ func GetServiceByHash(db sqlx.Queryer, hash []byte) (*Service, error) {
 func GetServiceByName(db sqlx.Queryer, machine, name string) (*Service, error) {
 	return GetService(db, squirrel.Eq{"name": name, "machine": machine})
 }
+
```

Comment:
Services is already defined under logxhost/service.go, so we don't need to redefine it.